cmd/in_memory/dispatcher/main.go

```diff
@@ -40,28 +38,12 @@ import (
 )
 
 var (
-	readTimeout  = 1 * time.Minute
-	writeTimeout = 1 * time.Minute
-
-	port                int
-	channelProvisioners listFlags
+	readTimeout         = 1 * time.Minute
+	writeTimeout        = 1 * time.Minute
+	port                = 8080
+	channelProvisioners = []string{"in-memory", "in-memory-channel"}
 )
 
-type listFlags []string
-
-func (l *listFlags) String() string {
-	return ""
-}
-func (l *listFlags) Set(value string) error {
-	*l = append(*l, value)
-	return nil
-}
-
-func init() {
-	flag.IntVar(&port, "sidecar_port", -1, "The port to run the sidecar on.")
```

Comment:
no longer need flags?

Reply:
Nope. Since the sidecar is no more generic and is specific to in-memory channel impl.